src/tasks/Microsoft.NET.Sdk.WebAssembly.Pack.Tasks/GenerateWasmBootJson.cs

```diff
@@ -423,7 +421,18 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
         }
 
         helper.ComputeResourcesHash(result);
-        JsonSerializer.Serialize(output, result, jsonOptions);
+
+        if (Path.GetExtension(OutputPath) == ".js")
```

Comment:
there is no change that passes `dotnet.boot.js` into this method in the PR, right ?

Reply:
Yes, we already had a property that allowed to change boot config file name. It is now set in Wasm.Build.Tests